docs/software/uenv/management.md

```diff
@@ -0,0 +1,364 @@
+- A **repository** (or repo) is a directory on the file system that contains downloaded uenv.
+  See the [repository][ref-uenv-repo] documentation for more information about repositories manage images.
+
+[](){#ref-uenv-find}
+## Finding and downloading uenv
+
+[](){#ref-uenv-image-find}
+### `uenv image find`
+The uenv images are in a registry can be queried using the `uenv image find` command:
+
+!!! example  "uenv image find"
+    ```console
+    $ uenv image find
+    uenv                       arch  system  id                size(MB)  date
+    cp2k/2024.1:v1             zen2  eiger   2a56f1df31a4c196   2,693    2024-07-01
+    cp2k/2024.2:v1             zen2  eiger   f83e95328d654c0f   2,739    2024-08-23
+    cp2k/2024.3:v1             zen2  eiger   7c7369b64b5fabe5   2,740    2024-09-18
+    editors/24.7:rc1           zen2  eiger   e5fb284962908eed   1,030    2024-07-18
+    editors/24.7:v2            zen2  eiger   4f0f2770616135b1   1,062    2024-09-04
+    julia/24.9:v1              zen2  eiger   0ff97a74dfcaa44e     539    2024-11-09
+    linalg/24.11:rc1           zen2  eiger   b69f4664bf0cd1c4     770    2024-11-20
+    linalg/24.11:v1            zen2  eiger   c11f6c85028abf5b     776    2024-12-03
+    linalg-complex/24.11:v1    zen2  eiger   846a04b4713d469b     792    2024-12-03
+    linaro-forge/24.0.2:v1     zen2  eiger   65734ce35494a5f5     313    2024-07-18
+    linaro-forge/24.1:v1       zen2  eiger   b65d7c85adfb317a     344    2024-11-27
+    netcdf-tools/2024:v1       zen2  eiger   e7e508c34cf40ccd   3,706    2024-11-14
+    prgenv-gnu/24.11:rc4       zen2  eiger   811469b00f030493     570    2024-11-21
+    prgenv-gnu/24.11:v1        zen2  eiger   0b6ab5fc4907bb38     572    2024-11-27
+    prgenv-gnu/24.7:v1         zen2  eiger   7f68f4c8099de257     478    2024-07-01
+    quantumespresso/v7.3.1:v1  zen2  eiger   61d1f21881a65578     864    2024-11-08
+    ```
+
+The output above lists all of the uenv that are available on the current system ([Eiger][ref-cluster-eiger] in this case).
+
+The search can be refined by providing a [label][ref-uenv-labels].
+
+??? example "Using labels to refine search"
+    ```bash
+    # find all uenv with name prgenv-gnu
+    uenv image find prgenv-gnu
+
+    # find all uenv with name and version prgenv-gnu/24.11
+    uenv image find prgenv-gnu/24.11
+
+    # find all uenv available for daint
+    uenv image find @daint
+
+    # find all prgenv-gnu uenv available on a cluster
+    uenv image find prgenv-gnu@daint
+
+    # find all uenv in the service namespace with name myenv
+    uenv image find service::myenv
+    ```
+
+!!! info
+    All uenv commands that take a [label][ref-uenv-labels] as an argument use the same flexible syntax [label descriptions][ref-uenv-labels-examples].
+
+[](){#ref-uenv-image-pull}
+### `uenv image pull`
+
+To download a uenv so that you can use it, the `uenv image pull` command will the registry to your local [repository][ref-uenv-repo].
```

Comment:
will... "download the image from"? not sure what sentence you might had in mind

```suggestion
To download a uenv so that you can use it, the `uenv image pull` command will ... the registry to your local [repository][ref-uenv-repo].
```